packages/api/lib/transformers/classifyTransformer.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const Transform = require('stream').Transform,
+  inherits = require('util').inherits,
+  moment = require('moment');
+
+const classifyTransformer = function (classifyTransformerOptions, streamOptions) {
+  if (!(this instanceof classifyTransformer)) {
+    return new classifyTransformer(classifyTransformerOptions, streamOptions);
+  }
+
+  if (!streamOptions) {
+    streamOptions = {};
+  }
+  streamOptions.decodeStrings = false;
+  streamOptions.objectMode = true;
+
```

Comment:
Consider intitialization of `now`, `now - 7 days` and `now - 30 days` variables upon transformer creation to avoid repeated initialization in `defineState`